### PR TITLE
Fetch latest changes from master and bump translate version

### PR DESCRIPTION
**Pull request by mziccard**
This PR fetches latest changes from master (required to make integration tests work). This also updates translate version to `0.2.8-SNAPSHOT`.

### CODE REVIEW

File: gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSubOptions.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
   private static final long serialVersionUID = 5640180400046623305L;
   private static final String PUBSUB_SCOPE = "https://www.googleapis.com/auth/pubsub";
   private static final Set<String> SCOPES = ImmutableSet.of(PUBSUB_SCOPE);
-  private static final String DEFAULT_HOST = "https://pubsub.googleapis.com";
+  private static final String DEFAULT_HOST = "pubsub.googleapis.com";
```
Comment (lesv) [inline gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSubOptions.java]: Why did you drop the https://?

**Comment by lesv** [comment]
One Q, otherwise LGTM

File: gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSubOptions.java (hunk 1 above)
Comment (mziccard) [inline gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSubOptions.java]: I dropped this as part of #1149. Pub/Sub is over gRPC so no need for `https://`.

**Pull request merged**